viewer/vue-client/.env.development

```diff
@@ -1,5 +1,9 @@
 VUE_APP_API_PATH=http://kblocalhost.kb.se:5000
-VUE_APP_AUTH_PATH=https://login.libris.kb.se/oauth/verify
+VUE_APP_VERIFY_PATH=https://login.libris.kb.se/oauth/verify
+VUE_APP_AUTHORIZE_PATH=https://login.libris.kb.se/oauth/authorize
+VUE_APP_REDIRECT_PATH=http://localhost:8080/katalogisering/login/authorized
+VUE_APP_CLIENT_ID=<get from developer>
```

Comment:
Maybe we need to move this file into an example-file? Since it is the one read by the application, but it is also tracked by git at the moment.

Reply:
Meaning: as soon as someone enters their client_id they will probably check it into git, with the current setup.

Reply:
Good point! Updated the PR.